app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -57,6 +68,9 @@ class HomeFragmentPresenter @Inject constructor(
     itemList.add(allTopicsViewModel)
     topicListAdapter = TopicListAdapter(activity, itemList, promotedStoryList)
 
+    profileIdFromIntent = activity.intent.getIntExtra(KEY_HOME_PROFILE_ID, 0)
```

Comment:
Default value should be `-1`. As `0` is related to admin profile.

Reply:
Updated to -1.